scripts/functions/clean_orphan_elements/clean_orphan_cases.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from scripts import constant
```

Comment:
```suggestion
import constant
```
Do not work with from script import constant by running in terminal